common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -458,114 +625,231 @@ void AutowareStatePanel::onMotion(const MotionState::ConstSharedPtr msg)
 
 void AutowareStatePanel::onMRMState(const MRMState::ConstSharedPtr msg)
 {
-  // state
-  {
-    QString text = "";
-    QString style_sheet = "";
-    switch (msg->state) {
-      case MRMState::NONE:
-        text = "NONE";
-        style_sheet = "background-color: #00FF00;";  // green
-        break;
+  IconState state;
+  QColor bgColor;
+  QString mrm_state = "MRM State | Unknown";
 
-      case MRMState::MRM_OPERATING:
-        text = "MRM_OPERATING";
-        style_sheet = "background-color: #FFA500;";  // orange
-        break;
+  switch (msg->state) {
+    case MRMState::NONE:
+      state = None;
+      bgColor = QColor("#84c2e6");
+      mrm_state = "MRM State | Inactive";
+      break;
 
-      case MRMState::MRM_SUCCEEDED:
-        text = "MRM_SUCCEEDED";
-        style_sheet = "background-color: #FFFF00;";  // yellow
-        break;
+    case MRMState::MRM_OPERATING:
+      state = Active;
+      bgColor = QColor("#84c2e6");
+      mrm_state = "MRM State | Active";
```

Comment:
```suggestion
      mrm_state = "MRM State | Operating";
```

Let's keep this as is to avoid confusion.